app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -4,8 +4,8 @@ module ManageIQ::Providers::Kubernetes::ContainerManagerMixin
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def raw_api_endpoint(hostname, port)
-      URI::HTTPS.build(:host => hostname, :port => port.presence.try(:to_i))
+    def raw_api_endpoint(hostname, port, path = '')
```

Comment:
We discussed that should be part of your change.
This patch does not add usage of a group client, but merely maintains backward compatibility.